spec/validation/configure_spec.rb

```diff
@@ -43,6 +43,11 @@
           type: 'integer',
           default: 10,
         },
+        boolean_true: {
+          question: 'Can I have a boolean true (/yes) default?',
+          type: 'boolean',
+          default: 'yes',
```

Comment:
I don't think we want to require `boolean` questions have their default specified as a `'yes'` or `'no'` string. Firstly this conversion is already performed within the `Configurator`, so requiring this here is unnecessary and also won't work once the default value is passed to the `Configurator` I believe.

Secondly, the default should be the default value which will be saved if no input is given, via whatever UI the user is using to perform the `configure`; in the case of `boolean` questions this will be `true`/`false` rather than `'yes'`/`'no'`. Doing this makes clear what this value is, and also means we can handle a default from a higher level saved answer file in the same way as a `configure.yaml`-defined default.

The use of `'yes'`/`'no'` to enter a default on the CLI can be considered a detail of this particular interface, used because accepting text is the simplest/standard way to get user input in a CLI. Other interfaces could accept different input to set a `boolean` value however, e.g. the GUI could use a check box.

Can we change this validation to validate for `true`/`false`?

Reply:
Will look into this. However it isn't ideal to fix it on this Pr but instead fix the issue off #185